pkg/scheduler/framework/plugins/nodevolumelimits/csi_test.go

```diff
@@ -397,7 +398,7 @@ func TestCSILimits(t *testing.T) {
 			ephemeralEnabled: true,
 			driverNames:      []string{ebsCSIDriverName},
 			test:             "ephemeral volume missing",
-			wantStatus:       framework.NewStatus(framework.Error, `looking up PVC test/abc-xyz: persistentvolumeclaim "abc-xyz" not found`),
+			wantStatus:       framework.AsStatus(errors.New(`looking up PVC test/abc-xyz: persistentvolumeclaim "abc-xyz" not found`)),
```

Comment:
ah, this is one of those tests using `reflect`. We will have to change this in a followup and this line will have to change.